src/main/java/io/github/moulberry/notenoughupdates/profileviewer/SkyblockProfiles.java

```diff
@@ -390,11 +392,13 @@ public JsonObject getOrLoadGuildInformation(Runnable runnable) {
 				updatingGuildInfoState.set(false);
 
 				if (jsonObject != null && jsonObject.has("success") && jsonObject.get("success").getAsBoolean()) {
-					if (!jsonObject.has("guild")) {
+					if (jsonObject.get("guild").isJsonNull()) {
+						isInGuild = false;
 						return null;
 					}
 
 					guildInformation = jsonObject.getAsJsonObject("guild");
+					System.out.println(guildInformation);
```

Comment:
Imagine keeping a debug log in production

Reply:
Can't imagine who would do such things